lib/src/runner/configuration/suite.dart

```diff
@@ -237,8 +237,10 @@ class SuiteConfiguration {
         runtimes: other._runtimes ?? _runtimes,
         includeTags: includeTags.intersection(other.includeTags),
         excludeTags: excludeTags.union(other.excludeTags),
-        tags: _mergeConfigMaps(tags, other.tags),
-        onPlatform: _mergeConfigMaps(onPlatform, other.onPlatform),
```

Comment:
Could `_mergeConfigMaps` have generic params and handle this conversion?

Reply:
yes, changed to do so.